src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -266,11 +266,23 @@ winrt::hstring Profile::ExpandedIconPath() const
 // - This profile's expanded background image path / the empty string.
 winrt::hstring Profile::ExpandedBackgroundImagePath() const
 {
+    WCHAR desktopWallpaper[MAX_PATH];

```

Comment:
nit: I would move this under the `DesktopWallpaper` else-if statement. That's the only place we need it.

Reply:
Resolved. Our school taught us to always declare any variables at start of functions. I prefer your way though, makes more sense.